src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Owoify.kt

```diff
@@ -30,11 +30,12 @@ import okhttp3.Request
 import java.io.File
 import java.util.*
 
-class Owoify : AbstractCommand(
+class Owoify : Command(
```

Comment:
Undo this, this breaks the OOP coding conventions.
Abstract classes are called `Abstract{Name}`
Interfaces are called `I{Name}`
and implementations should be called `{Name}{BaseClassName}`, unless there is one implementation for an interface then you can just append `Impl`